sunpy/coordinates/utils.py

```diff
@@ -384,3 +385,41 @@ def get_rectangle_coordinates(bottom_left, *, top_right=None,
             top_right = top_right.frame
 
     return bottom_left, top_right
+
+
+def solar_angle_equivalencies(observer):
+    """
+    Return the equivalency to convert between arcsec on the Sun and km.
+
+    Parameters
+    ----------
+    observer : `~astropy.coordinates.SkyCoord`
+        Observer for which equivalency is calculated.
+
+    Returns
+    -------
+    equiv : equivalency function that can be used as keyword `equivalencies` for astropy unit conversion.
+
+    Examples
+    --------
+    >>> earth_observer = get_body_heliographic_stonyhurst("earth", "2013-10-28")
+    >>> distance_in_km = 725*u.km
+    >>> distance_in_km.to(u.arcsec, equivalencies=solar_angle_equivalencies(earth_observer))
+    <Quantity 1.00603718 arcsec>
+    """
+
+    if not isinstance(observer, (SkyCoord, BaseCoordinateFrame)):
+        raise TypeError(
+            "Invalid input, observer must be of type SkyCoord or BaseCoordinateFrame.")
+
+    obstime = observer.obstime
+    sun_coord = get_body_heliographic_stonyhurst("sun", time=obstime, observer=observer)
+    sun_observer_distance = sun_coord.separation_3d(observer).to_value(u.m)
+
+    equiv = [(u.radian,
+              u.meter,
+              lambda x: np.tan(x)*sun_observer_distance,
+              lambda x: np.arctan(x/sun_observer_distance))]
+
+    log.info(f"The equivalency uses an approximate conversion using tangent, this does not approximate well for large angles.")
```

Comment:
Don't make this a logger statement.  Add this instead as a note in the docstring.

Also, as far as the wording goes, it's not so much an "approximation" as it is a "definition".  You just need to make people aware which choice this function makes, so that they can evaluate whether their definition is the same or at least close to the same for small angles.

Reply:
I have updated this, see if it is satisfactory.

Reply:
I think this looks good to me.  Once its ok for @ayshih

Reply:
Yeah, **it** definitely appears as a definition. I think unless we are dealing with a function with multiple possible output scenarios then the logger statement is indeed undesirable.